.github/workflows/kernel-release.yml

```
@@ -0,0 +1,121 @@
+
+      - name: Build kernel
+        run: |
+          docker run --rm \
+            --volume "${{ github.workspace }}:/work" \
+            --workdir /work \
+            smolvm-kernel-builder \
+            /work/kernel/scripts/build.sh ${{ matrix.arch }}
+
+      - name: Record artifact metadata
+        id: meta
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:6:57: Double quote to prevent globbing and word splitting

(shellcheck)